tendermint/vue
# feat: Use CRUD itemName for create button

## Description & Context

We did not correctly display the name of the module in the CRUD component. By default, we always displayed "post" and not item name;

## How To Test

1. Open the [Data.view](https://github.com/tendermint/vue/blob/develop/packages/template/src/views/Data.vue) template;
3. The button should display the correct element name from the Crud component;

## Linked issue

Closes #202
